libevmasm/AssemblyItem.h

```diff
@@ -59,6 +59,9 @@ enum AssemblyItemType
 	ReturnContract, ///< Returns new container (with auxiliary data filled in) to be deployed
 	RelativeJump, ///< Jumps to relative position accordingly to its argument
 	ConditionalRelativeJump, ///< Same as RelativeJump but takes condition from the stack
+	CallF, ///< Calls a function under function ID. Function is returning.
+	JumpF, ///< Calls a function under function ID. Function is non-returning.
+	RetF, ///< Returns from returning function to the caller.
```

Comment:
This does not seem entirely accurate. According to EIP-6206 it's possible to jump to a returning function as well. I'd phrase the distinction differently:

```suggestion
	CallF, ///< Jumps to a returning EOF function, adding a new frame to the return stack.
	JumpF, ///< Jumps to a returning or non-returning EOF function without changing the return stack.
	RetF, ///< Returns from an EOF function, removing a frame from the return stack.
```